docs/getting-started/quickstart-sms.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: quickstart-managed-scans
+append_help_link: true
+title: "Quickstart: Managed Scans"
+hide_title: true
+description: Set up Semgrep Managed Scans when you sign in to Semgrep for the first time.
+tags:
+  - Quickstart
+  - Semgrep AppSec Platform
+---
+
+# Quickstart for Semgrep Managed Scans
+
+Semgrep Managed Scans (beta) is the fastest method to scan repositories at scale with Semgrep. Instead of adding Semgrep to your CI/CD pipeline, which requires a configuration file for each repository, Semgrep handles the scan process for all of the repositories you add to it.
+
+## Supported source code managers
+
+- Semgrep Managed Scans is available for **GitHub-hosted (GitHub.com) and GitHub Enterprise Server** plans.
+
+## Prerequisites
+
+- Admin access to your GitHub organization.
+- **Read access** to your code must be granted to Semgrep. This is achieved through a private Semgrep app that you create and register yourself. See [Managed Scans > Security](/deployment/managed-scanning#security) for more information on how Semgrep handles your code.
+
+## Add repositories to Semgrep Managed Scans
+
+<!-- vale off -->
+<!-- Our in-product text reads "repos" -->
+
+1. Navigate to [Semgrep AppSec Platform](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub**. Follow the on-screen prompts to [grant Semgrep the necessary permissions](/deployment/checklist/#permissions) and proceed.
+1. Provide the **Organization display name** you'd like to use, then click **Create new organization**.
+1. When asked **Where do you want to scan?** click **GitHub**.
+1. Follow the steps in the **Connect GitHub to Semgrep** page. These steps install a public GitHub app, which handles PR comments, and a private GitHub app, which handles code access. You are able to select which repositories these apps have access to, and have full control over removing them or revoking their permissions.
+1. Click **Set up projects**. You are taken to the **Enable Managed Scans for GitHub repos** page.
+1. Select all the repositories you want to add to Semgrep Managed Scans for scanning.
+1. Click **Enable Managed Scans**. You are taken to the **Projects** page as your scans begin.
+
+<!-- vale on -->
+
+You have finished setting up a Semgrep managed scan.
+
```

Comment:
I think the subsequent bullet points needs a header to distinguish them from the setup instructions; maybe something like "Considerations?"